org.eclipse.xtext/META-INF/MANIFEST.MF

```diff
@@ -8,49 +8,57 @@ Bundle-Vendor: Eclipse Xtext
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-11
 Export-Package: org.eclipse.xtext,
- org.eclipse.xtext.build;x-friends:="org.eclipse.xtext.testing,
+ org.eclipse.xtext.build;
+  x-friends:="org.eclipse.xtext.testing,
    org.eclipse.xtext.ide,
    org.eclipse.xtext.junit4,
    org.eclipse.xtext.tests,
-   org.eclipse.xtend.core.tests",
+   org.eclipse.xtend.core.tests,
+   org.eclipse.xtext.builder.standalone.tests,
+   org.eclipse.xtend.standalone,
+   org.eclipse.xtext.builder.standalone",
  org.eclipse.xtext.common,
  org.eclipse.xtext.common.parser.packrat;x-friends:="org.eclipse.xtext.generator",
  org.eclipse.xtext.common.parser.packrat.consumers;x-friends:="org.eclipse.xtext.generator",
  org.eclipse.xtext.common.services,
  org.eclipse.xtext.conversion,
  org.eclipse.xtext.conversion.impl,
- org.eclipse.xtext.debug;x-friends:="org.eclipse.xtext.builder,
-   org.eclipse.xtext.ide.tests,
-   org.eclipse.xtext.xbase",
+ org.eclipse.xtext.debug;x-friends:="org.eclipse.xtext.builder,org.eclipse.xtext.ide.tests,org.eclipse.xtext.xbase",
```

Comment:
no newline after each bundle?

Reply:
As above: I added them with the MANIFEST editor.